File: calc-code-pl-general-ivonarum/calc.py

# Python Simple Calculator
print("Python Calculator") 
# aby zdefiniować funkcję używamy słowa kluczowego 'def' (w JS 'function')
def add_py(a, b): # parametry w nawiasie po nazwie funkcji (tak samo jak w JS)
    return a + b # słowo kluczowe RETURN aby utworzyć c.iało f.unkcji (w JS c.f. jest w klamrze)
def subtract_py(a, b): # funkcja substract dwuparametrowa (parametry po przecinku)
    return a - b # funkcja odejmowania utworzona za pomocą znaku '-'
def multiply_py(a, b): # funkcja multiply dwuparametrowa
    return a * b # funkcja mnożenia utworzona znakiem '*'
def divide_py(a, b): # funkcja divide dwuparametrowa
    return a / b # funkcja dzielenia utworzona znakiem '/'

# JAVA, JS, PY tak samo wywołuje się funkcje: nazwa funkcji i wartość parametrów w nawiasie
# print i w nawiasie kod wywołania funkcji od razu wyświetla jej wynik na konsoli 
print(add_py(10, 5)) # wyświetla wynik funkcji add (10+5)
print(subtract_py(10, 5)) # wyświetla wynik funkcji substract (10-5)
print(multiply_py(10, 5)) # wyświetla wynik funkcji multiply (10*5)
print(divide_py(10, 5)) # wyświetla wynik funkcji divide (10/5)

File: calc-code-pl-general-ivonarum/calc.js

// JavaScript Simple Calculator
console.log("JavaScript Calculator"); 
// aby zdefiniować funkcję używamy słowa kluczowego 'function' (w PY 'def')
// parametry w nawiasie po nazwie funkcji (tak samo jak w PY)
// słowo kluczowe RETURN aby utworzyć ciało funkcji
// ciało funkcji jest w klamrze (w PY tylko wcięcie)
function addJs(a, b) { return a + b; } // funkcja dwuparametrowa-dodawanie znakiem '+'
function subtractJs(a, b) { return a - b; } // funkcja odejmowania utworzona za pomocą znaku '-'
function multiplyJs(a, b) { return a * b; } // funkcja mnożenia utworzona znakiem '*' 
function divideJs(a, b) { return a / b; } // funkcja dzielenia utworzona znakiem '/' 

// JAVA, JS, PY tak samo wywołuje się funkcje: nazwa funkcji i wartość parametrów w nawiasie
// console.log (w PY 'print') i w nawiasie kod wywołania funkcji od razu wyświetla jej wynik na konsoli 
console.log(addJs(10, 5)); // wyświetla wynik funkcji add (10+5)
console.log(subtractJs(10, 5)); // wyświetla wynik funkcji substract (10-5)
console.log(multiplyJs(10, 5)); // wyświetla wynik funkcji multiply (10*5)
console.log(divideJs(10, 5)); // wyświetla wynik funkcji divide (10/5)
